fineract-provider/src/main/java/org/apache/fineract/useradministration/api/UsersApiResource.java

```diff
@@ -64,7 +64,7 @@
      * {@link AppUserData}.
      */
     private final Set<String> RESPONSE_DATA_PARAMETERS = new HashSet<>(Arrays.asList("id", "officeId", "officeName", "username",
-            "firstname", "lastname", "email", "allowedOffices", "availableRoles", "selectedRoles", "staff"));
+            "firstname", "lastname", "email", "allowedOffices", "availableRoles", "selectedRoles", "staff", "currentPassword"));
```

Comment:
is this secure and safe? From what (little) I understand, this will expose any user's current password in the API (to other accounts with admin permissions, I guess, but still) which perhaps doesn't seem like such a great idea? Passwords usually are "write only", and then verified, but never returned back out of a system.

@awasum @avikganguly01 @vishwasbabu or anyone else reading this, care to chime in if I misunderstand this and it's OK or if this is No Go?

Reply:
I think this should be avoided.